layouts/docs/list.html

```diff
@@ -1,3 +1,45 @@
 {{ define "main" }}
-{{ .Render "content" }}
-{{ end }}
\ No newline at end of file
+<div class="td-content">
+	<h1>{{ .Title }}</h1>
+	{{ with .Params.description }}<div class="lead">{{ . | markdownify }}</div>{{ end }}
+	{{ .Content }}
+<div class="section-index">
```

Comment:
What if this div (or the contents of the div) was a partials so that it could be easily overridden/personalized?

Reply:
As in the section index div? Sure, though we've already given them three potential options for customizing it and it's already most of the page.

I'll have a go and see how it looks.

Reply:
I think that's actually nicer! Thanks for suggestion.